test/op_shape_test.cpp

```diff
@@ -1819,6 +1819,33 @@ TEST_CASE(pad_dyn_shape1)
     expect_shape(output, migraphx::make_op("pad", {{"pads", {0, 0, 1, 1, 0, 0, 1, 1}}}), input);
 }
 
+TEST_CASE(pointwise_no_module)
+{
+    migraphx::shape input{migraphx::shape::float_type, {0}, {0}};
+    throws_shape(migraphx::make_op("pointwise"), input);
+}
+
+TEST_CASE(pointwise_no_input)
+{
+    migraphx::program p;
+    auto* mm = p.get_main_module();
+    migraphx::module m;
+    std::vector<migraphx::instruction_ref> args{};
+    migraphx::shape output{migraphx::shape::float_type, {1}, {0}};
+    auto l = m.add_literal(migraphx::literal(output, {1}));
+    m.add_return({l});
+    mm->add_instruction(migraphx::make_op("pointwise"), args, {&m});
+}
+
+TEST_CASE(pointwise_no_output)
```

Comment:
```suggestion
TEST_CASE(pointwise_no_input_args)
```

Reply:
This is intentional so we dont have any output shape assigned for the module going into pointwise. since this block in pointwise

```
      if(pm->get_output_shapes().size() != 1)
            MIGRAPHX_THROW("submodule should have only one output.");
        // if pointwise op doens't have any input args (const-foldable) then just return output
        // shape
```